mapie/classification.py

```diff
@@ -342,68 +343,98 @@ def _check_proba_normalized(
         )
         return y_pred_proba
 
+    def _adaptive_mask_quantile(
+        self,
+        arr: ArrayLike,
+        quantile: float
+    ) -> NumpyInt:
+        """
+        Adapt the quantile of the mask to the case where the minimum value
+        is greater than the quantile.
+
+        Parameters
+        ----------
+        arr : ArrayLike
+            1D-array given by the apply_along_axid method. It contains the
+            values of one observation.
+        quantile : float
+            quantile with which we want to compare.
+
+        Returns
+        -------
+        int
+            Index of the last included label for a specific observation and
+            and specific quantile.
+        """
+        quantile = np.maximum(quantile, np.min(arr))
+
+        return np.argmax(np.ma.masked_greater(arr, quantile + EPSILON))
+
     def _get_last_score_included(
         self,
-        y_pred_proba_sorted_cumsum: ArrayLike,
+        y_pred_proba_cumsum: ArrayLike,
         include_last_label: Optional[Union[bool, str]]
-    ) -> Optional[ArrayLike]:
+    ) -> ArrayLike:
         """
         Return the index of the last included sorted probability
         depending if we included the first label over the quantile
         or not.
 
         Parameters
         ----------
-        y_pred_proba_sorted_cumsum : ArrayLike
-            Sorted cumsumed probabilities.
+        y_pred_proba_cumsum : ArrayLike
```

Comment:
Toujours préciser `ArrayLike of shape (n_samples, n_classes)`

VALABLE POUR TOUTES LES DOCSTRINGS DE CE MODULE

Reply:
done